BlueMapCore/src/main/webroot/js/libs/BlueMap.js

```diff
@@ -515,17 +515,17 @@ export default class BlueMap {
 
 				resolve(object);
 			}, () => {
-			}, reject);
+			}, (msg)=>{console.log(msg)});
```

Comment:
Leftover debug? Or is there a reason for this change? :D

Reply:
My fault